build.gradle.kts

```diff
@@ -8,12 +8,14 @@ buildscript {
         classpath(ClassPaths.ANDROID_GRADLE)
         classpath(ClassPaths.KOTLIN_GRADLE)
         classpath(ClassPaths.HILT_GRADLE)
+        classpath(ClassPaths.GOOGLE_SERVICES)
     }
 }
 
 plugins {
     id(Plugins.SPOTLESS) version Versions.SPOTLESS
     id(Plugins.DETEKT) version Versions.DETEKT
+    id("org.jetbrains.kotlin.android") version "1.8.0" apply false
```

Comment:
See above line for how we're writing plugins.

Reply:
Changed it to appropriate format.